package/src/index.js

```diff
@@ -1,9 +1,15 @@
 import React from "react";
 
+
+const render = (Linker, Config) => {
+    return Config.map((link, index) => Linker[link.module](index, link.params.text, render(Linker, link.children)))
```

Comment:
notable changes from our meeting: send the raw params object instead of just the "text" attribute